### ISSUE TITLE
God / Bundler / RVM

### ISSUE THREAD

**Issue by JDutil** [mojombo/god#65]
I'm having an extremely difficult time figuring out how to make the combination of God / Bundler / RVM all play nicely together.  I do have a setup that is working & running though.  My assumption is that it is the fact that god is run with sudo as root rather than as my deploy user.  I've tried following rvm's guide on using a wrapper, but it simply resulted in the same issue.

The issue I have is that anytime I now attempt to change a gem in my Gemfile it breaks god.  I will receive errors that it cannot find the new gem in my log file.  I have the gem installed under both the system default ruby, and for each rvm rubies & gemset combination.   I've eliminated all of the rubies not being used so I am now down to just the ubuntu system default 1.8.7, and the rvm installed 1.9.2.  The gems are installed under the system default ruby, the ruby 1.9.2 global gemset, and the ruby 1.9.2 custom gemset.  I can't get god to find the gems though even though my start/stop commands are being run through bundle exec...

I ran into this issue once before, and after hours of tedious uninstalling rvm, reinstalling, and server reboots etc.. I manage to somehow mysteriously get it to work that I really wasn't sure what solved it.  Now I've run into this again, and after trying to basically uninstall / reinstall rvm all over again and rebooting the server with the gem installed under every ruby / gemset combination on the system I still run into gem not found errors.

Basically it runs fine with the current production Gemfile I have, but I cannot possibly update any gems without it causing god to break from being unable to find the new gems.  This is all despite the fact that they are installed both during capistrano deployment, and then I will even manually install them under every ruby / gemset combination on the system.

Does anyone else use god with bundler & rvm that could help shed light on this issue?  Is there anyway to possibly run god as my deploy user?  Or could someone help me debug this issue?  I can provide any relevant information that would help track down the root cause of this if it would help just let me know.
